tests/phpunit/tests/test-media.php

```diff
@@ -80,38 +79,6 @@ public function test_attachment_fields_to_edit() {
 		$this->assertFalse( isset( $fields['language'] ) );
 	}
 
-	/**
-	 * @since 3.1 Since the language and translations are updated through a previous AJAX call, we'd rather not perform an unnecessary update now.
-	 */
-	public function test_attachment_fields_to_save() {
```

Comment:
Why don't we need this test anymore?

Reply:
This test is checking that updating a media (but not its language) does not update the langue nor the translations. But `PLL_Translated_Object::set_language` and `PLL_Translatable_Object::set_language` are taking care of this right now. See: https://github.com/polylang/polylang/blob/29854c72ddaf357772bc12b1e7627236bf839f8d/include/translated-object.php#L86-L90 and https://github.com/polylang/polylang/blob/3.4-dev/include/translatable-object.php#L167-L169
And since the test is checking that `set_language` and `save_translations` are not called through a mock, this is becoming obsolete. Indeed, these function are now called anyway.

Reply:
This was indeed the purpose of this test. Can we add alternative test(s) that will adapt this check to our new way to update languages?

Reply:
We started a list of tests to implement, and it's added into it. Should I create the test in this PR?